docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -0,0 +1,470 @@
+---
+approvers:
+- mtaufen
+- dawnchen
+title: Reconfigure a Node's Kubelet in a Live Cluster
+---
+
+{% capture overview %}
+{% include feature-state-alpha.md %}
+As of Kubernetes 1.8, the new
+[Dynamic Kubelet Configuration](https://github.com/kubernetes/features/issues/281)
+feature is available in alpha. This allows you to change the configuration of
+Kubelets in a live Kubernetes cluster via first-class Kubernetes concepts.
+Specifically, this feature allows you to configure individual Nodes' Kubelets
+via ConfigMaps.
+
+**Warning:** All Kubelet configuration parameters may be changed dynamically,
+but not all parameters are safe to change dynamically. This feature is intended
+for system experts who have a strong understanding of how configuration changes
+will affect behavior. No documentation currently exists which plainly lists
+"safe to change" fields, but we plan to add it before this feature graduates
+from alpha.
+{% endcapture %}
+
+{% capture prerequisites %}
+- A live Kubernetes cluster with both Master and Node at v1.8 or higher must be
+running, with the `DynamicKubeletConfig` feature gate enabled and the Kubelet's
+`--dynamic-config-dir` flag set to a writeable directory on the Node.
+This flag must be set to enable Dynamic Kubelet Configuration.
+- The kubectl command-line tool must be also be v1.8 or higher, and must be
+configured to communicate with the cluster.
+{% endcapture %}
+
+{% capture steps %}
+
+## Reconfiguring the Kubelet on a Live Node in your Cluster
+
+### Basic Workflow Overview
+
+The basic workflow for configuring a Kubelet in a live cluster is as follows:
+
+1. Write a YAML or JSON configuration file containing the
+Kubelet's configuration.
+2. Wrap this file in a ConfigMap and save it to the Kubernetes control plane.
+3. Update the Kubelet's correspoinding Node object to use this ConfigMap.
```

Comment:
Typo, `corresponding`

Reply:
Thanks, sent typo fixes for the versions containing this doc:
- 1.10: #7640
- 1.9: #7641
- 1.8: #7642